instagram-app/src/components/CommentSection/CommentContainer.js

```diff
@@ -37,12 +51,21 @@ componentWillReceiveProps(newProps) {
             comments: [...this.state.comments, newComment],
             comment: ''
         })
+    }
 
+    handleLike = e => {
+        const likes = this.state.likes + 1
+        this.setState({
+            likes : likes
+        })
+        localStorage.setItem('likes', this.state.likes)
```

Comment:
how do you know the difference of likes from one post to another if you are just setting likes in localstorage for everyone of them. Try incrementing the likes on the post and then every time the posts change save them to local storage.